net-misc/youtube_dl/youtube_dl-2018.02.08.recipe

```diff
@@ -26,20 +26,18 @@ REQUIRES="
 	"
 
 BUILD_REQUIRES="
+	haiku_devel
 	"
-
 BUILD_PREREQUIRES="
+	cmd:find
+	cmd:make
 	"
 
-BUILD()
-{
-	sed -i 's/\/usr//' youtube-dl-$portVersion.py
-	mv youtube-dl-$portVersion.py youtube-dl
-	chmod a+x youtube-dl
-}
```

Comment:
The port is a bit more complicated. Looks like this port is a perfectly valid python module that uses setuptools. This basically means we would have to use this ugly "pattern"
https://github.com/haikuports/haikuports/blob/99b5ce0a6ffec38d8db4304ec4fc2f58108a9253/dev-python/docutils/docutils-0.14.recipe#L31

Why should we further complicate this port? So that youtube-dl could be [embedded][0] to a python program.

[0]: https://github.com/rg3/youtube-dl/tree/2018.02.08#embedding-youtube-dl

Reply:
That's a good idea! Thanks for the suggestion!
(I'll try to implement this before tomorow.)